### DISCUSSION TITLE
Microsoft Access write issue

### DISCUSSION THREAD

**Discussion by zulkefliaris** [dbeaver/dbeaver#12836] [General]
I am having issues inserting, updating and deleteing records in Microsoft Access database from a Windows 10 machine. The mdb file is stored in a network drive.  If I copy the exact file to my local disk, all works fine. I have tried disabling the Windows 10 machine firewall but the issue still persist. Both Dbeaver and Java is on the latest vesion. I have another Windows 7 machine running the same version of Dbeaver and Java but I do not face the same issue. Both machines login to the network drive using the same account. Please advice the cause of the problem.

**Comment by kugelpunk** [comment]
Hello @zulkefliaris, I am planning to do the same. Have you identified the problem already somehow?
